src/index.js

```diff
@@ -1,33 +1,242 @@
+  const completed = false;
+  const Index = activityArray.length + 1;
+  const Object = new MyConstructor(description, completed, Index);
+  activityArray.push(Object);
+  saveTaskstoLocalStorage();
+  AddToScreen();
+};
+
+const submitForm = (event) => {
+  if (inputArea.value !== '') {
+    event.preventDefault();
+    AddtoLocalStorage();
+    saveTaskstoLocalStorage();
+    AddToScreen();
+  }
+  Form.reset();
+};
+
+Form.addEventListener('submit', submitForm);
+// double click enter icon
+const clickToAddNewTask = () => {
+  if (inputArea.value !== '') {
+    AddtoLocalStorage();
+    saveTaskstoLocalStorage();
+    AddToScreen();
+    Form.reset();
+  }
+};
+enter.addEventListener('dblclick', clickToAddNewTask);
+
+// check box change
+const handleCheckboxchange = (event) => {
+  const checkbox = event.target;
+  const listItem = checkbox.parentNode.parentNode;
+  const taskId = listItem.getAttribute('id');
+  activityArray[taskId].completed = checkbox.checked;
+  saveTaskstoLocalStorage();
+  if (checkbox.checked) {
+    listItem.querySelector('.text').classList.add('completed');
+  } else {
+    listItem.querySelector('.text').classList.remove('completed');
+  }
+  saveTaskstoLocalStorage();
+};
+
+activitySection.addEventListener('change', handleCheckboxchange);
+
+const loadActivityArrayFromLocalStorage = () => {
+  const activityArrayString = localStorage.getItem('activities');
+  if (activityArrayString) {
+    return JSON.parse(activityArrayString);
+  }
+  return [];
+};
+
+const activityArray1 = loadActivityArrayFromLocalStorage();
+
+activityArray1.forEach((task, index) => {
+  const listItem = document.getElementById(index);
+  if (task.completed) {
+    listItem.querySelector('.text').classList.add('completed');
+  }
+});
+
+// deleteing a task
+
+const DeleteTask = (e) => {
+  if (e.target.classList.contains('trash')) {
+    const listItem = e.target.parentNode.parentNode;
+    const taskIndex = listItem.getAttribute('id');
+    // const taskIndex = Array.from(listItem.parentNode.children).indexOf(listItem) - 1;
+    activityArray.splice(taskIndex, 1);
+    reIndex();
+    saveTaskstoLocalStorage();
+    AddToScreen();
+  }
+};
+activitySection.addEventListener('click', DeleteTask);
+// deleting all checked tasks
+
+const deleteCheckedTasks = () => {
+  activityArray = activityArray.filter((MyConstructor) => !MyConstructor.completed);
+  saveTaskstoLocalStorage();
+  AddToScreen();
+};
+// button for dleting all checked tasks
+const cLearALL = () => {
+  deleteCheckedTasks();
+  reIndex();
+  saveTaskstoLocalStorage();
+  AddToScreen();
+};
+clearAll.addEventListener('click', cLearALL);
+// refresh function
+// const reFreshPage = (e) => {
+//   if (e.target.classList.contains('refresh')) {
+//     reFresh.setAttribute('animation', 'spin');
+//     window.location.reload();
+//     window.addEventListener('load', () => {
+//       reFresh.setTimeout(() => reFresh.classList.remove('animation'), 4500);
+//     });
+//   }
+// };
```

Comment:
- [ ] Please kindly remove all commented code from your codebase as it does not follow best practices and corrupts the codebase.